repo-structure.md

```diff
@@ -0,0 +1,169 @@
+
+```
+
+The structure above is then processed when you build the collection, to generate a Docker image for the stack, along with tar files of each of the templates and pipelines, which can then all be stored/referenced in a local or public appsody repo. Refer to the section on [Building and Testing Stacks](/docs/stacks/build-and-test.md) for more details. The appsody CLI can then access such a repo, to use the stack to initiate local development.
+
+## Summary of files within the stack and collection source and user directory structure
+
+### stack.yaml
+
+The `stack.yaml` file in the top level directory defines the different attributes of the stack and which template the stack should use by default. See the example below:
+
+```bash
+name: Sample Application Stack   # concise one line name for the stack
+version: 0.1.0                   # version of the stack
+description: sample stack to help creation of more appsody stacks # free form text explaining more about the capabilities of this stack and various templates
+license: Apache-2.0              # license for the stack
+language: nodejs                 # programming language the stack uses
+maintainers:                     # list of maintainer(s) details
+  - name: John Smith
+    email: example@example.com
+    github-id: jsmith
+default-template: my-template-1  # name of default template
+```
+
+### Collection.yaml
+
+The `collection.yaml` file in the top level directory defines the different attributes of the collection and which container image and pipeline the collection should use by default. See the example below:
+
+```bash
+default-image: java-microprofile # name of the default containier image - reference into the images element below
+default-pipeline: default        # name of the default pipeline - reference to the pipeline in the directory structure
+images:                          # list of container images
+- id: java-microprofile
+  image: $IMAGE_REGISTRY_ORG/java-microprofile:0.2
+```
+
+### README
+
+The top level directory must contain a `README.md` markdown file that describes the contents of the collection and how it should be used.
+
+### LICENSE
+
+The `image` directory must contain a `LICENSE` file.
+
+### app-deploy.yaml
+
+The `app-deploy.yaml` is the configuration file for deploying an Appsody project using the Appsody Operator. For more information about specifics, see [Appsody Operator User Guide](https://github.com/appsody/appsody-operator/blob/master/doc/user-guide.md).
+
+### Dockerfile-stack
+
+The `Dockerfile-stack` file in the `image` directory defines the foundation stack image, and a set of environment variables that specify the desired behaviour during the rapid local development cycle. It also defines what is exposed from the host machine to the container during this mode.
+
+Environment variables can be set to alter the behaviour of the CLI and controller (see [Appsody Environment Variables](/docs/stacks/environment-variables.md)).
```

Comment:
see [Appsody Environment Variables](/docs/stacks/environment-variables.md)). is a 404 too

Reply:
Fixed